R/derive_param_exist_flag.R

```diff
@@ -7,19 +7,19 @@
 #'
 #' @param dataset Input dataset
 #'
-#'   The variables specified for `subject_keys` and the `PARAMCD` variable are
+#'   The variables specified for `by_vars` and the `PARAMCD` variable are
 #'   expected.
 #'
-#' @param dataset_adsl ADSL input dataset
+#' @param dataset_ref (ADSL) input dataset
 #'
-#'   The variables specified for `subject_keys` are expected. For each subject
-#'   (as defined by `subject_keys`) from the specified dataset (`dataset_adsl`),
+#'   The variables specified for `by_vars` are expected. For each subject
```

Comment:
Should be "by group" instead of "subject".

Reply:
Not sure if I get this correctly. 
Do you mean `group` as a unique group defined by the variables in `by_vars`?
I have tried to replace in the upcoming commit, but not sure if this is what you meant. 
`subject` is also used in the description of the other parameters (`condition`, `true_value`,`false_value`& `missing_value`). Should these be changed as well?

Reply:
Yes, by "by group" I mean the groups as defined by `by_vars`. These may be not subjects but something else. E.g., subjects and visits like in the Hy's Law vignette.
Thus subject should be replaced in the description of all arguments.

Reply:
Will update again tomorrow!